heron/packing/src/java/com/twitter/heron/packing/Container.java

```diff
@@ -23,50 +28,94 @@
  */
 public class Container {
 
-  //Resources currently used by the container.
-  private long usedRam;
-  private double usedCpuCores;
-  private long usedDisk;
+  private HashSet<PackingPlan.InstancePlan> instances;
 
   //Maximum resources that can be assigned to the container.
   private long maxRam;
```

Comment:
Instead of storing each part of the resource passed, can we just store the resource itself?